crates/macro-support/src/parser.rs

```diff
@@ -377,11 +377,17 @@ impl<'a> ConvertToAst<BindgenAttrs> for &'a mut syn::ItemStruct {
             .unwrap_or(self.ident.to_string());
         let is_inspectable = attrs.inspectable().is_some();
         let getter_with_clone = attrs.getter_with_clone().is_some();
+        let transparent = self.attrs.iter().any(|attr| {
+            matches!(attr.style, AttrStyle::Outer)
+                && attr.path.is_ident("repr")
+                && attr.tokens.to_string() == "(transparent)"
```

Comment:
I think that this match should be done with something more structured than calling `.to_string` on the tokens